plugins/single_plugins/place.cpp

```diff
@@ -36,6 +36,8 @@ class wayfire_place_window : public wf::plugin_interface_t
             std::string mode = placement_mode;
             if (mode == "cascade")
                 cascade(view, workarea);
+            else if (mode == "max")
```

Comment:
Since 'max' doesn't really make sense for a placement type, maybe this should be 'maximize' to make it clear.

Reply:
I agree. I think "maximum" also makes sense, no? I changed it to this in the following commit.

Reply:
changed to maximize